nuprojs/Microsoft.ServiceFabric.Services.Remoting/Microsoft.ServiceFabric.Services.Remoting.nuproj

```diff
@@ -98,5 +98,8 @@
       <Dependency Include="Microsoft.ServiceFabric.FabricTransport.Internal">
         <Version>[$(NugetPkg_Version_Microsoft_ServiceFabric_FabricTransport_Internal)]</Version>
       </Dependency>
+	  <Dependency Include="System.Diagnostics.DiagnosticSource">
+		<Version>[5.0.0.0]</Version>
```

Comment:
https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges
Do not use exact version match. Also take dependency on lowest possible supported version

Reply:
Made the change